test/SparqlParserTest.cpp

```diff
@@ -922,7 +922,7 @@ TEST(ParserTest, testSolutionModifiers) {
   {
     auto pq = SparqlParser(
                   "SELECT DISTINCT ?x SCORE(?x) ?y WHERE \t {?x :myrel ?y}\n"
-                  "ORDER BY ASC(?y) DESC(SCORE(?x)) LIMIT 10 OFFSET 15")
+                  "ORDER BY ASC(?y) LIMIT 10 OFFSET 15")
```

Comment:
This test had to be changed. Do you prefer having a TODO there to readd `SCORE` or just have the test commented out?

Reply:
I would suggest:
* Comment it out completely
* Still add the TODO that we need to figure out the cleanest way to add the textScore back into standard-conforming SPARQL.